setup/setup-lokole.sh

```diff
@@ -644,7 +644,19 @@ main() {
   fi
 }
 
-main 2>&1 | tee --append "\${sync_logfile}"
+main_with_retry() {
+  attempts=0
+  timeout=5
+  until "$(attempts -ge 5)"
+  do
+    main 2>&1 | tee --append "\${sync_logfile}" && break
+    attempts="$(attempts+1)"
```

Comment:
This would be more standard Bash (and therefore more cross-platform compatible) by using the Bash [arithmetic expression syntax](https://www.tldp.org/LDP/abs/html/arithexp.html):

```suggestion
    attempts="$((attempts+1))"
```

It's worth double-checking that this also works in `sh` and not just `bash`, but I'm pretty sure it should work.